catfeeder-machine/golog.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+    "os"
+    "strings"
+    "sync"
+    "time"
+)
+
+type Golog struct {
+    Lock        sync.Mutex
+    Fpath       *os.File
+    Prefix      string
+}
+
+const LogFilename string = "/tmp/cat-feeder.log"
+
+/**
+ * @brief:  Checks to see if the log file is present from a previous
+ *          execution. If true, roll over log with timestamp. Then
+ *          open a new log file.
+ *
+ * @return: nil on success, else error
+ **/
+func InitGolog() error {
```

Comment:
The description says that this function creates a new log file after it rolls over a previous log file but it doesn't open/create a log file in `InitGolog()`. It open/creates one in `OpenGolog()`.

Reply:
Updated